### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by sudokufan**
Help me with my navbar please!

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en-GB">
+
+<head>
+	<link rel="stylesheet" type="text/css" href="styles.css" />
+	<link rel="stylesheet" type="text/css" href="normalize.css" />
```
Comment (jdselby24) [inline about.html]: link normalize.css before you link your styles to avoid future issues

File: contact.html [hunk 2]
```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en-GB">
+
+<head>
+	<link rel="stylesheet" type="text/css" href="styles.css" />
+	<link rel="stylesheet" type="text/css" href="normalize.css" />
```
Comment (jdselby24) [inline contact.html]: link normalize.css before you link your styles to avoid future issues

File: index.html [hunk 3]
```diff
@@ -1,10 +1,25 @@
 <!DOCTYPE html>
 <html lang="en-GB">
+
 <head>
 	<link rel="stylesheet" type="text/css" href="styles.css" />
 	<link rel="stylesheet" type="text/css" href="normalize.css" />
```
Comment (jdselby24) [inline index.html]: link normalize.css before you link your styles to avoid future issues

File: portfolio.html [hunk 4]
```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en-GB">
+
+<head>
+	<link rel="stylesheet" type="text/css" href="styles.css" />
+	<link rel="stylesheet" type="text/css" href="normalize.css" />
```
Comment (jdselby24) [inline portfolio.html]: link normalize.css before you link your styles to avoid future issues

File: styles.css [hunk 5]
```diff
@@ -1,3 +1,75 @@
 * {
 	box-sizing: border-box;
+	font-family: Helvetica;
```
Comment (jdselby24) [inline styles.css]: add failover font e.g sans-serif, for if Helvetica is not supported on device

File: styles.css [hunk 6]
```diff
@@ -1,3 +1,75 @@
...
+
+nav li {
+	list-style: none;
+	float: left;
+	width: 25%;
+	color: #C2BAC2;
+}
+
+a:link {
+  text-decoration: none;
```
Comment (jdselby24) [inline styles.css]: BLOAT: consider applying text-decoration: none to the 'a' tag generally rather than for each state

File: about.html (hunk 1 above)
Comment (sudokufan) [reply about.html]: good note

File: styles.css (hunk 5 above)
Comment (sudokufan) [reply styles.css]: _font-family: Helvetica; sans-serif;_
like this?

File: styles.css (hunk 6 above)
Comment (sudokufan) [reply styles.css]: YES I knew there'd be a simpler way

File: styles.css (hunk 5 above)
Comment (jdselby24) [reply styles.css]: yep 

`                          (__)
                           (oo)
  /---------------\/
 / |                      ||
*  ||------------||
   OO            OO
      li-moo-cow`

**Pull request merged**